docs/manage-security-verification-for-wog-account.md

```diff
@@ -0,0 +1,33 @@
+# Set up security verification for Whole-of-Government (WOG) account
```

Comment:
pls remove these changes as this is now a step in the onboarding PO into TechPass